frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1196,7 +1196,7 @@ function ReaderHighlight:lookup(selected_text, selected_link)
 
     -- if we extracted text directly
     if selected_text.text and self.hold_pos then
-        self.ui:handleEvent(Event:new("LookupWord", selected_text.text, false, word_boxes, self, selected_link))
+        self.ui:handleEvent(Event:new("LookupWord", selected_text.text, false, word_boxes, self, selected_link, nil, {selected_text.pos0, selected_text.pos1}))
```

Comment:
Can we not pass this new stuff down the cascading calls - because it feels a bit random in this context.
At any time, I think any other thing (even your plugin if there's a document loaded, so check that self.ui.highlight is not nil) can access `self.ui.highlight.selected_text`(.pos0...) to get more info.
It's not unusual for other bits of code to go pick what they need directly in the other modules via self.ui.thatothermodule.

(I'm not even sure why we pass these args to `lookup(selected_text, selected_link)` - it's always self.selected_text.)